resources/templates/index.html

```diff
@@ -27,7 +27,7 @@ <h1>
     Launch
   </button>
   {% else %}
-  <button class="link-button" data-link-url="{{ reverse_url('setup', 'mainnet') }}" id="setup-button" disabled>
+  <button class="link-button" data-link-url="{{ reverse_url('setup', 'mainnet', None) }}" id="setup-button" disabled>
```

Comment:
One suggestion: We could call the create wallet method here. Then we could get rid of the if branch in the setupHandler